<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPG Assistant</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="buttonTurnOrderList">
     <property name="text">
      <string>&amp;Turn order list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonNPCGenerator">
     <property name="text">
      <string>&amp;NPC Generator</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonCalendar">
     <property name="text">
      <string>&amp;Calendar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVersion">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
